src/Directory.Build.props

```diff
@@ -2,9 +2,22 @@
     <PropertyGroup>
         <PackageProjectUrl>http://fable.io</PackageProjectUrl>
         <RepositoryUrl>https://github.com/fable-compiler/Fable.git</RepositoryUrl>
-        <PackageLicense>https://raw.githubusercontent.com/fable-compiler/Fable/master/LICENSE</PackageLicense>
-        <PackageIconUrl>https://raw.githubusercontent.com/fable-compiler/Fable/master/docs/img/fable_logo.png</PackageIconUrl>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageIcon>fable_logo.png</PackageIcon>
         <PackageTags>fsharp;fable;javascript;f#;js</PackageTags>
         <Authors>Alfonso García-Caro Núñez</Authors>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+
+        <!-- For SourceLink. See: https://github.com/dotnet/sourcelink#using-source-link-in-net-projects -->
+        <PublishRepositoryUrl>true</PublishRepositoryUrl>
+        <EmbedUntrackedSources>true</EmbedUntrackedSources>
+        <IncludeSymbols>true</IncludeSymbols>
+        <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     </PropertyGroup>
+    <ItemGroup>
+        <None Include="..\..\fable_logo.png" Pack="true" PackagePath="\" />
```

Comment:
It seems the fable_logo.png icon is missing from this repo, we should probably add it. It'd be also great if the directory is not relative (I guess we can use the MsBuildThisFileDirectory constant) so we can easily copy the file to other repos without depending on a specific structure.

Reply:
Yep, it's included in the PR.

Reply:
Instead of using a relative link, I propose to use `$(MSBuildThisFileDirectory)`

If find it easier to read and it will works for all the packages in the repository independently of their level.

For example, in Fable.Form I do it like that for the license file

```xml
<PropertyGroup>
    <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
</PropertyGroup>
<ItemGroup>
    <None Include="$(MSBuildThisFileDirectory)\LICENSE.txt" Pack="true" PackagePath="$(PackageLicenseFile)" />
</ItemGroup>
```

Reply:
Done.